DOC-1719: Replace TOC and edit descriptions for databases/import-export index page (#2257)
RedisLabs/redislabs-docs: content/rs/databases/import-export/_index.md

    /rs/databases/import-export/,
]
---
You can [import]({{< relref "/rs/installing-upgrading/customize-user-and-group.md" >}}), [export]({{< relref "/rs/databases/import-export/export-data.md" >}}),
or [backup]({{< relref "/rs/databases/import-export/schedule-backups.md" >}})
files of a specific Redis Enterprise Software database to restore data.

## [Import data]({{< relref "/rs/installing-upgrading/customize-user-and-group.md" >}})

> Import export or backup files of a specific Redis Enterprise Software database to restore data. You can either import from a single file or from multiple files, such as when you want to import from a backup of a clustered database.

Reviewer: You also might be able to just remove the second sentence here.
```suggestion
Import data from a backup or another Redis database. You can import from a single file or multiple files, such as when you want to import a backup of a clustered database.
```

## [Export data]({{< relref "/rs/databases/import-export/export-data.md" >}})

> Export data to import it into a new database or to make a backup. Export to a local mount point, to [a URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) using FTP/SFTP, or on cloud provider storage.

Reviewer: The first sentence seems a bit confusing and since there are other articles about importing and backups included on this entry page, I think it would be okay to remove that part.
```suggestion
Export data from a Redis Enterprise database to a local mount point, an FTP or SFTP server, or cloud provider storage.
```
